Makefile

```diff
@@ -1,30 +1,39 @@
 PATH := node_modules/.bin/:$(PATH)
```

Comment:
Is this line unnecessary now?

Reply:
I'm wondering whether we should leave it so that we will continue to support [native installations](https://prairielearn.readthedocs.io/en/latest/installingNative/)?

Reply:
Yeah, good point...